src/lib/types/model.ts

```diff
@@ -196,7 +196,7 @@ export interface IProjectOverview {
     createdAt: Date | undefined;
     stats?: IProjectStats;
     mode: ProjectMode;
-
+    featureLimit?: number;
```

Comment:
can we make it always defined and return number | null?

Reply:
Using` featureLimit?: number; `allows you to omit the featureLimit property when creating an object or defining a variable of that type. This can be useful in scenarios where the featureLimit may or may not be provided.